psamm/datasource/sbml.py

```diff
@@ -701,14 +702,20 @@ def _get_flux_bounds(self, r_id, model, flux_limits, equation):
             else:
                 lower_str = text_type(-model.get_default_flux_limit())
         else:
-            lower_str = text_type(flux_limits[r_id][0])
+            lower = flux_limits[r_id][0]
+            if lower % 1 == 0 and lower != -model.get_default_flux_limit():
```

Comment:
The default flux limit here is usually 1000.0 as far as I could see, so without this there would be duplicates.